gbpservice/nfp/proxy_agent/proxy/proxy.py

```diff
@@ -184,7 +186,9 @@ def recv(self):
         return None
 
     def send(self, data):
-        self._socket.send(data)
+        self._socket.setblocking(1)
+        self._socket.sendall(data)
```

Comment:
this was just a hunch while going over the components w.r.t perf activity that it could happen that proxy doesnt send up all the data to over the cloud.. need to verify this separately.